_pages/how-we-work/tools/tock.md

```diff
@@ -66,20 +66,25 @@ You must not bill for the following activities because these are indirect costs
 
 ### How many hours am I expected to bill per week?
 
-The majority (80%) of your time at 18F will be dedicated to your billable project work. If it's not explicitly an indirect cost, it's project work.
+If you’re an individual contributor or a lead, you’re expected to spend at least  80% of your time  on billable project work. (Directors and supervisors have different billability expectations.)
 
-To keep projects moving forward as planned in our SOWs, based on the number of full (8-hour) days you work in a given week, you need to spend at least the following number of hours on project work:
+That leaves up to 20% of your time to be spent on non-billable work. There are two types of non-billable work, both of which are used to manage the running of 18F itself: GSA-mandated non-billable work and 18F non-billable work. 
 
-Days in the office for the week | Hour Expectation
---- | ---
-5 days | At least 32\* billable hours<br>Up to 8 hours non-billable
-4 days | At least 28\* billable hours<br>Up to 4 hours non-billable
-3 days | At least 22\* billable hours<br>Up to 2 hours non-billable
-2 days | At least 16\* billable hours<br>Up to 0 hours non-billable
-1 days | At least 8\* billable hours<br>Up to 0 hours non-billable
+GSA-mandated non-billable work includes time spent in HRLinks, mandatory OLU trainings, IDPs, SF-182s, annual reviews, and so forth. These are the things we don’t have a choice but to spend time on.
 
+18F non-billable work includes time spent in guilds and working groups, team coffees, chapter meetings, business development, conference attendance, and anything that contributes to the running of 18F as an organization. 
 
```

Comment:
What do you think of adding a mention here of non-billable approved internal projects that can always use help, like Tock, for example?

Reply:
added